tests/python_tests/data/models.py

```diff
@@ -80,4 +80,8 @@ def get_gguf_model_list():
             "gguf_model_id": "Qwen/Qwen2.5-0.5B-Instruct-GGUF",
             "gguf_filename": "qwen2.5-0.5b-instruct-q4_0.gguf"
         },
+        {
```

Comment:
Its weight is ~1.9GB. May be we should add rather to nightly validation? @Wovchena, please advice.
Or create a tiny version of it if possible?

Reply:
Hi @Wovchena, thanks for your suggestion, create a dummy model on-the-fly is good idea, while not easy to figure out how to set proper gguf tensor and metadata per model. 

Instead, I create a tiny dummy gguf model w/ single transformer block with minimal size. Now I update the CI test with dummy model.